src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personName } = useParams();
+
+  const changingEndOfLink = (currentPerson: Person) => {
```

Comment:
read `person.slug` instead - check the people.json file
![image](https://github.com/mate-academy/react_people-table-basics/assets/86923089/78e6fcaa-1da7-4b08-9b84-236dc2c6eb4e)

Reply:
and get rid of this funcrion and of `selectedPersonName`